File: stepik-java-api/src/main/java/org/stepik/api/actions/StepikLessonsAction.java
package org.stepik.api.actions;

import org.jetbrains.annotations.NotNull;
import org.stepik.api.client.StepikApiClient;
import org.stepik.api.queries.lessons.StepikLessonsGetQuery;

/**
 * @author meanmail
 */
public class StepikLessonsAction extends StepikAbstractAction {
    public StepikLessonsAction(@NotNull StepikApiClient stepikApiClient) {
        super(stepikApiClient);
    }

    @NotNull
    public StepikLessonsGetQuery get() {
        return new StepikLessonsGetQuery(this);
    }
}

File: stepik-java-api/src/main/kotlin/org/stepik/api/actions/StepikLessonsAction.kt
package org.stepik.api.actions

import org.stepik.api.client.StepikApiClient
import org.stepik.api.queries.lessons.StepikLessonsGetQuery

class StepikLessonsAction(stepikApiClient: StepikApiClient) : StepikAbstractAction(stepikApiClient) {
    
    fun get(): StepikLessonsGetQuery {
        return StepikLessonsGetQuery(this)
    }
}
